code/modules/ruins/objects_and_mobs/ash_walker_den.dm

```diff
@@ -7,8 +7,8 @@
 	icon_state = "ash_walker_nest"
 	icon_living = "ash_walker_nest"
 	faction = list("ashwalker")
-	health = 200
-	maxHealth = 200
+	health = 400
+	maxHealth = 400
```

Comment:
I've already explained it. This PR doesn't make them overpowered. Far from it. It just makes playing them more feasible. 

All that's left is for you to either actually play it (and it seems like you won't, probably because you don't know how) or go be a piece of shit somewhere else. Otherwise your opinion about this carries no weight.